Tab-sets for Manifest V2/V3 (#2101)
MicrosoftDocs/edge-developer: microsoft-edge/extensions-chromium/store-policies/ada-addendum.md

> <!-- ====================================================================== -->

Author: Maybe change the title of this article from
App Developer Agreement Addendum for Microsoft Edge program users
to
Addendum to Microsoft Edge Add-ons store developer policies

and swap the TOC order of the two docs to become:
Microsoft Edge Add-ons store developer policies
Addendum to Microsoft Edge Add-ons store developer policies

Author: I believe that the link destination is incorrect.  The present article is not an addendum to the store policies, but to the ADA.  All evidence is consistent with that except the actual link destination.  I'm entering a Suggestion to fix the link presentation & destination.

## Exhibit G: Terms and Conditions for Microsoft Edge Extensions

> These Terms and Conditions for Microsoft Edge Extensions (the "Extensions Addendum") is a supplement to the terms and conditions of the App Developer Agreement, the current version of which can be found at [https://go.microsoft.com/fwlink/p/?LinkID=221922](developer-policies.md).  This Addendum applies if you have submitted or are making available Microsoft Edge Extensions (each, an "Extension") through Microsoft Edge Add-ons.  Except as expressly modified by this Extensions Addendum, all of the terms and conditions in the App Developer Agreement, which is incorporated by reference herein, apply to the offering and distribution of your Microsoft Edge Extensions.  Except where expressly modified by this Addendum, the terms and conditions that apply to an "Application" or "App" under the App Developer Agreement will apply to an Extension as defined in this Addendum and all references to "Store" or "Microsoft Store" under the App Developer Agreement will apply to Microsoft Edge Add-ons as defined in this Addendum.

Author: @sonakshimore @ashishnirvana1 - 

2x, the link text gives the URL for the Windows app actual "App Developer Agreement", and the body text of present page says "App Developer Agreement" about 10x.  

But, clicking the link takes you instead to the local doc titled "Microsoft Edge Add-ons store developer policies".   Which of the two docs is meant?

Each spot that says "App Developer Agreement" in this page needs to be examined, maybe change all to "Microsoft Edge Add-ons store developer policies".

Author: As I posted on another comment:
I believe that the link destination is incorrect. The present article is not an addendum to the store policies, but to the ADA. All evidence is consistent with that except the actual link destination. I'm entering a Suggestion to fix the link presentation & destination.

> These Terms and Conditions for Microsoft Edge Extensions (the "Extensions Addendum")  is a supplement to the terms and conditions of the Windows [App Developer Agreement](https://go.microsoft.com/fwlink/p/?LinkID=221922) together with the parallel [Microsoft Edge Add-ons store developer policies](developer-policies.md).

Author: ```suggestion
These Terms and Conditions for Microsoft Edge Extensions (the "Extensions Addendum")  is a supplement to the terms and conditions of the Windows [App Developer Agreement](https://go.microsoft.com/fwlink/p/?LinkID=221922) and applies along with [Microsoft Edge Add-ons store developer policies](developer-policies.md).
```

*  This Addendum applies if you have submitted or are making available Microsoft Edge Extensions (each, an "Extension") through Microsoft Edge Add-ons.

> *  Except as expressly modified by this Extensions Addendum, all of the terms and conditions in the App Developer Agreement,<!--todo: doc title/link?--> which is incorporated by reference herein, apply to the offering and distribution of your Microsoft Edge Extensions.

Author: ```suggestion
*  Except as expressly modified by this Extensions Addendum, all of the terms and conditions in the App Developer Agreement, which is incorporated by reference herein, apply to the offering and distribution of your Microsoft Edge Extensions.
```

> *  Except where expressly modified by this Addendum, the terms and conditions that apply to an "Application" or "App" under the App Developer Agreement<!--todo: doc title/link?--> will apply to an Extension as defined in this Addendum and all references to "Store" or "Microsoft Store" under the App Developer Agreement<!--todo: doc title/link?--> will apply to Microsoft Edge Add-ons as defined in this Addendum.

Author: ```suggestion
*  Except where expressly modified by this Addendum, the terms and conditions that apply to an "Application" or "App" under the App Developer Agreement will apply to an Extension as defined in this Addendum and all references to "Store" or "Microsoft Store" under the App Developer Agreement will apply to Microsoft Edge Add-ons as defined in this Addendum.
```

> *  Except where expressly modified by this Addendum, the terms and conditions that apply to an "Application" or "App" under the App Developer Agreement<!--todo: doc title/link?--> will apply to an Extension as defined in this Addendum and all references to "Store" or "Microsoft Store" under the App Developer Agreement<!--todo: doc title/link?--> will apply to Microsoft Edge Add-ons as defined in this Addendum.

Author: ```suggestion
*  Except where expressly modified by this Addendum, the terms and conditions that apply to an "Application" or "App" under the App Developer Agreement will apply to an Extension as defined in this Addendum and all references to "Store" or "Microsoft Store" under the App Developer Agreement will apply to Microsoft Edge Add-ons as defined in this Addendum.
```

> 1.  DEFINITIONS.  Capitalized terms not defined herein will have the meanings provided in the App Developer Agreement.<!--todo: doc title/link?-->

Author: ```suggestion
1.  DEFINITIONS.  Capitalized terms not defined herein will have the meanings provided in the App Developer Agreement.
```

    1.  "Certification Requirements" means the technical, functional, content, and other policy requirements provided by Microsoft (at [https://go.microsoft.com/fwlink/?linkid=2104222](developer-policies.md), or another location(s) specified by Microsoft) for Extensions offered through Microsoft Edge Add-ons.

    1.  "Microsoft Edge Add-ons" means a Microsoft owned or operated platform, however named, through which Microsoft Edge Extensions may be offered to or acquired by Customers.

> 1.  SUBMISSION, CERTIFICATION, AND DISTRIBUTION OF APPS.  Section 3 of the App Developer Agreement<!--todo: doc title/link?--> is hereby deleted and replaced by the following:

Author: ```suggestion
1.  SUBMISSION, CERTIFICATION, AND DISTRIBUTION OF APPS.  Section 3 of the App Developer Agreement is hereby deleted and replaced by the following:
```

    1.  Updates to Extensions.  You may submit updates to Extensions for Certification and distribution through Microsoft Edge Add-ons.  Those updates are subject to all of the requirements of this Agreement.  You understand that end users may receive updates automatically.  You may not add any new functionality to your Extension via an update which requires written consent from the Customer without first providing notice to the Customer and obtaining any consents as may be required by law in the markets where you choose to distribute your Extension.

    1.  Initial Certification.  Microsoft will test each Extension (including any updates) you submit for compliance with this Agreement (which includes the applicable Certification Requirements) and other policies made available to you (if any).  Microsoft will not make any Extension available through Microsoft Edge Add-ons unless and until the Extension is Certified.

> 1.  AFFIRMATIVE DECLINE NOT REQUIRED.  Notwithstanding Section 4(i) of the App Developer Agreement<!--todo: doc title/link?-->, you aren't required to affirmatively decline participation in Microsoft Store for Business and Microsoft Store for Education.  Your Extension will only be available through the Microsoft Edge Add-ons.

Author: ```suggestion
1.  AFFIRMATIVE DECLINE NOT REQUIRED.  Notwithstanding Section 4(i) of the App Developer Agreement, you aren't required to affirmatively decline participation in Microsoft Store for Business and Microsoft Store for Education.  Your Extension will only be available through the Microsoft Edge Add-ons.
```

> 1.  APP PRICING, PAYMENTS, TRANSACTIONS AND TAXES.  Section 6 of the App Developer Agreement<!--todo: doc title/link?--> does not apply to Extensions and is hereby deleted in its entirety.

Author: ```suggestion
1.  APP PRICING, PAYMENTS, TRANSACTIONS AND TAXES.  Section 6 of the App Developer Agreement does not apply to Extensions and is hereby deleted in its entirety.
```

<!-- ====================================================================== -->
## Exhibit G: Terms and Conditions for Microsoft Edge Extensions

These Terms and Conditions for Microsoft Edge Extensions (the "Extensions Addendum")  is a supplement to the terms and conditions of the Windows [App Developer Agreement](https://go.microsoft.com/fwlink/p/?LinkID=221922) together with the parallel [Microsoft Edge Add-ons store developer policies](developer-policies.md).

*  This Addendum applies if you have submitted or are making available Microsoft Edge Extensions (each, an "Extension") through Microsoft Edge Add-ons.

> 

Reviewer: @Reezaali  @mikehoffms  - The link change to ADA seems like a bug that and fixing that should not require a CELA approval. Are there any other major changes to this document apart from the link to ADA? If not, I think this does not require CELA approval, we should go ahead.

Author: The present 2101 is superseded by a couple newer PRs, see equivalent comment from Ashish in new PR
https://github.com/MicrosoftDocs/edge-developer/pull/2273#discussion_r1007961529

In the new PR that affects this article, there are no changes to this article other than this link fix to make the link actually go to the page that the link text claims that the link goes to.
